src/modules/musig/main_impl.h

```diff
@@ -561,14 +600,24 @@ int secp256k1_musig_partial_sig_verify(const secp256k1_context* ctx, const secp2
     /* Multiplying the messagehash by the musig coefficient is equivalent
      * to multiplying the signer's public key by the coefficient, except
      * much easier to do. */
-    secp256k1_musig_coefficient(&mu, session->pk_hash, signer->index);
+    secp256k1_musig_coefficient(&mu, session->keygen_aux.pk_hash, signer->index);
     secp256k1_scalar_mul(&e, &e, &mu);
 
     if (!secp256k1_pubkey_load(ctx, &rp, &signer->nonce)) {
         return 0;
     }
+    memcpy(&pubkey_tmp, pubkey, sizeof(pubkey_tmp));
```

Comment:
A structure assignment is better style here (and I think in general, reading possible struct padding bytes is UB).

Reply:
structure assignment is better style, fixed